splunk/src/main/java/com/splunk/HttpService.java

```diff
@@ -37,8 +37,16 @@
 public class HttpService {
     // For debugging purposes
     private static final boolean VERBOSE_REQUESTS = false;
-    public static boolean useTLS=false;

```

Comment:
Let's add this to the CHANGELOG:

Breaking change: `HttpService.useTLS` flag removed, please use `HttpService.setSslSecurityProtocol()` to set a specific SSL/TLS implementation or else TLS 1.2 is used by default for Java 1.8.